src/Microsoft.AspNet.Hosting/Startup/ConfigureDelegate.cs

```diff
@@ -20,10 +20,7 @@ public ConfigureBuilder(MethodInfo configure)
 
         public MethodInfo MethodInfo { get; }
 
-        public Action<IApplicationBuilder> Build(object instance)
-        {
-            return builder => Invoke(instance, builder);
-        }
+        public Action<IApplicationBuilder> Build(object instance) => builder => Invoke(instance, builder);
```

Comment:
C# 6.0 ftw, yeah its pretty snazzy looking since it gets rid of my personal pet peeve, the bloated naked { } lines for one liners :)